grafana/data_source_organization_preferences.go

```diff
@@ -0,0 +1,82 @@
+package grafana
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func DatasourceOrganizationPreferences() *schema.Resource {
+	return &schema.Resource{
+		Description: `
+* [Official documentation](https://grafana.com/docs/grafana/latest/administration/manage-organizations/)
+* [HTTP API](https://grafana.com/docs/grafana/latest/developers/http_api/preferences/#get-current-org-prefs)
+`,
+		ReadContext: dataSourceOrganizationPreferencesRead,
+		Schema: map[string]*schema.Schema{
```

Comment:
You can reuse your resource's schema with this function: https://github.com/grafana/terraform-provider-grafana/blob/master/grafana/schema.go#L23-L45